### ISSUE TITLE
Linux/mingw64 (cross-compiler) fortran

### ISSUE THREAD

**Issue by ForsteriteFoe** [modflowpy/pymake#14]
Please find attached a hacked version of the pymake.py file to use the mingw64 toolchain for compiling a windows x64.exe version of MODFLOW using Linux.  Here is the general reference for how the mingw64 gfortran differs from standard gfortran (it also has options for the 32-bit versions):

[https://sourceforge.net/p/mingw-w64/wiki2/UsingLinuxBinaries/#using-the-cross-compilers-to-generate-windows-32-bit-and-64-bit-binary-executables](url)

Basically, in addition to the prefix for both the gfortran and gcc, in the linker phase needs the -static option, as well as replace the -lc option with either -m64 or -m32.

Don't currently have a clean patch for you.

[pymake_w64.py.txt](https://github.com/modflowpy/pymake/files/977169/pymake_w64.py.txt)

**Comment by ForsteriteFoe** [comment]
I'll see if sometime after work, I can put this all together in a nice clean package that is less of a hack.  Probably won't get to this for a week or two.
